src/enums/viewState.ts

```diff
@@ -88,6 +88,14 @@ export const DismissedViewState = makeEnum({
 export type DismissedViewState =
   typeof DismissedViewState[keyof typeof DismissedViewState];
 
+/**
+ * Consent Manager view state for the Interactive Advertising Bureau (IAB) Tech Lab's Global Privacy Platform (GPP)
+ */
+export const GPPViewState = makeEnum({
```

Comment:
```suggestion
export const IABViewState = makeEnum({
```

Reply:
GPP really doesn't have view states

Reply:
Cool. I actually had IABViewState at first, but thought GPP was more precise since TCF is part of GPP (right?)

Reply:
eh, i would describe GPP as just the consent string encoding that can encode a TC string.

GPP does not have any specifications/opinions about UI